interactions/api/gateway/client.py

```diff
@@ -111,6 +150,7 @@ def __init__(
             "timeout": 60,
             "autoclose": False,
             "compress": 0,
+            "proxy": _proxy if isinstance(_proxy, ProxyConfig) else None,
```

Comment:
FYI (or a tip/suggestion/advice)
the proxy arg here have to either be string or None, not proxyconfig objects
(+, if you have an BasicAuth object, you can do "proxy_auth": <your auth object> if it exists)

Reply:
(yeah I noticed that after I pushed)
I'll correct a lot later when I'm done with work